server/db/migrations/20211230165827_harden-letter_versions-table.js

```diff
@@ -0,0 +1,44 @@
+const tableName = 'letter_versions'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.alterTable(tableName, function (table) {
+      // Rename columns
+      table.renameColumn('campaignid', 'campaign_id')
```

Comment:
Sweet! After this update, we may need to update this [line](https://github.com/ProgramEquity/amplify-back-end/blob/main/server/routes/api/letter_versions.js#L9) cc: @BeeXiong

Reply:
@BeeXiong any thoughts here?

Reply:
Updated in commit https://github.com/ProgramEquity/amplify-back-end/pull/147/commits/27aa8632b01ed8058caa16a5cc76b37ab95692a6